src/dev/riscv/lupv.hh

```diff
@@ -45,8 +46,8 @@ class LupV : public Platform
   public:
     Clint *clint;
     LupioRTC *lupioRTC;
-    Plic *plic;
-    int uartIntID;
+    int lupioTTYIntID;
+    Plic *pic;
```

Comment:
I'm not sure if this is what was requested. The pic is still a type `Plic` because I wasn't sure if I was supposed to create a generic class named `Pic` from which the `plic` can be derived?

Reply:
It'll get replaced by the lupio-pic eventually, so it's not a big deal.